catalog/app/containers/Auth/messages.js

```diff
@@ -30,6 +30,36 @@ export default defineMessages({
     defaultMessage: 'Authenticating...',
   },
 
+  // SSO
+  ssoGoogleUse: {
+    id: `${scope}.SSO.Google.use`,
+    defaultMessage: 'Use Google Account',
```

Comment:
Should probably be something like "Sign in with Google" -though I'll let @akarve decide.

Also, it appears in uppercase cause of our button styles - but, in my opinion, it doesn't look good on the Google button.

Reply:
re:wording: when someone uses sso, they may be either signed-in or signed-up, so i preferred more neutral wording (asana uses this btw).
as for button style: it's consistently used throughout the app and i think it looks okayish ([official branding guide](https://developers.google.com/identity/branding-guidelines) says to use regular casing tho)

Reply:
OK as Use Google Account